### PR TITLE
Changlog 1.0.4.2

### PR DESCRIPTION
**Pull request by NicolasDorier**

### CODE REVIEW

**Comment by NicolasDorier** [comment]
Ping @pavlenex there is also new language stuff right?

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## 1.0.4.2
+
+### Bug fixes
+* With LND above 0.9, invoices were immediately transitioning as partially paid. (@r0ckstardev)
+* Successful payjoin in P2SH-P2WPKH would result in partially paid invoice (@MrKukks)
```
Comment (Kukks) [inline Changelog.md:7]: overPaid not partially

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+
+### Improvements
+
+* Checkout dark theme improvements (@dennisreimann #1508)
+* Show warning when create a hot wallet when you are not admin of the server (@MrKukks)
+
+### Greenfield
+
+* Expose an health check endpoint without authentication (@dennisreimann)
+* Very primitive create/read/update store endpoints (@MrKukks)
```
Comment (Kukks) [inline Changelog.md:20]: update still in PR. 
Delete is also avauilable

**Comment by pavlenex** [comment]
@NicolasDorier For people whose GitHub I couldn't match with 100% certainty or it did not exist, I lined to Transifex profile as a credit. 

Here you go:

**New invoice checkout languages added:**
- Bulgarian (Bulgaria) (bg_BG) @doynovbps
- Danish (Denmark) (da_DK) @Berlelund
- Norwegian (no) [@devenia](https://www.transifex.com/user/profile/devenia/)
- Persian (fa) [@firildakh](https://www.transifex.com/user/profile/firildakh/)
- Romanian (ro) [@BTCfactura](https://www.transifex.com/user/profile/BTCfactura/)
- Slovak (Slovakia) (sk_SK) [@MSedivy](https://www.transifex.com/user/profile/MSedivy/)
- Zulu (zu) [@kpangako](https://www.transifex.com/user/profile/kpangako/)

**Updated translation for checkout invoice:**
- Arabic (Ar) @kemoantemo
- Bosnian (Bosnia and Herzegovina) (bs_BA) @Ruxiol
- Danish (Denmark) (da_DK) @Berlelund
- German (Germany) (de_DE)[@andhans](https://twitter.com/andhans_jail)
- Greek (Greece) (el_GR) @kaloudis
- Spanish (Spain) (es_ES) @RzeroD
- Hindi(hi) @blockbitmedia
- Indonesian (id) @anditto 
- Polish (pl) [@kodxana](https://www.transifex.com/user/profile/kodxana/)
- Portuguese (Pt_pt) [MarcosMe](@https://www.transifex.com/user/profile/MarcosMe/)
- Turkish (tr) [@https://www.transifex.com/user/profile/efecini/](https://www.transifex.com/user/profile/efecini/)

**Pull request merged**